src/ch/ch_monitor.h

```diff
@@ -30,6 +30,7 @@
 #define URL_VMM_SHUTDOWN "vmm.shutdown"
 #define URL_VMM_PING "vmm.ping"
 #define URL_VM_CREATE "vm.create"
+#define URL_VM_RESIZE "vm.resize"
```

Comment:
Minor nit: This might be better suited in the next commit

Reply:
Ah indeed, fixing.